### PR TITLE
update go version

### PR DESCRIPTION
**Pull request by whitewum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,12 +17,16 @@ This repository contains the Nebula Graph Console for Nebula Graph 2.0. Nebula G
...
 
+    > NOTE: Go version provided with apt on ubuntu is usually "outdated".  
+
     Run the following command to examine if Go is installed on your machine.
 
     ```bash
     $ go version
     ```
 
+    The version should be newer than 1.16.2.
```
Comment (jievince) [inline README.md:28]: Why is 1.16.2?

Comment (jievince) [reply README.md:28]: 1.13 is enough

Comment (whitewum) [reply README.md:28]: because it is current goversion

how about update to a new version for future use?

Comment (jievince) [reply README.md:28]: go 1.13 is enough for console use， even if upgrading to a new version of console in the future

Comment (whitewum) [reply README.md:28]: got！

**Review by jievince — APPROVED** [review]
LGTM

**Pull request merged**